### DISCUSSION TITLE
Want to help in building this can i ? And want to study about this ...

### DISCUSSION THREAD

**Discussion by 2006Kartik** [rabilrbl/jiotv_go#343] [Q&A] [answered]
I am a student currently in first year of Bachelor of engineering in Computer engineering ... I dont know anything about this go language as i only know is go language is used by google ... I want to learn from the source code of this ... is there any developer who can teach me how this stuff works... i know basics of javascript and some java stuff and html .... i want to learn coding something which is out of just building calculators .... i want to learn more about this ... and now i have soo much time to learn .. tell me if someone cam teach me reply me i will reply you for contact details

**Comment by rabilrbl** [comment, marked as answer]
Hi,

Welcome to the project! It's great to see your enthusiasm for learning Go and contributing to open source. However, it's important to approach learning with a mindset of self-initiative.

I would recommend starting with some online resources to learn the basics of Go. YouTube has many excellent tutorials, and websites like Go by Example, the Go Tour, and the official Go documentation are great places to start.

Once you have a good grasp of the basics, you can explore our codebase to see how things are structured and understand the project's architecture. Try to read the code, run it, and even experiment by making small changes.

If you have specific questions or run into issues, feel free to ask here, and the community (https://telegram.me/jiotv_go_chat) will be more than happy to help. However, it's essential to do some self-learning first to build a foundation.

Happy coding, and we look forward to your contributions!
